css/styles.css

```diff
@@ -47,62 +44,276 @@ nav {
   padding: 0;
 }
 
-.container-1 {
-  width: 100%;
-  height: 30px;
+hr {
+  width: 85%;
+  margin: 10px auto 10px auto;
+  padding: 0;
+  border-bottom: solid 2px #dfe1e6;
 }
 
-.container-2 {
+.poppins {
   font-family: Poppins, sans-serif;
-  margin-left: 0;
-  margin-right: 48px;
+}
+
+/* Toolbar */
+.container-2 {
+  margin-left: 27px;
+  margin-right: 37px;
+  margin-top: 10px;
   display: flex;
-  align-items: center;
   justify-content: space-between;
+  align-items: center;
 }
 
+/* Section Headline */
 .headline {
   margin: 20px, 0, 20px, 0;
-  font-family: Poppins, sans-serif;
   color: #172b4d;
-  width: 375px;
+  width: 100%;
   display: flex;
   flex-direction: column;
 }
 
 .logo-txt {
-  font-family: Poppins, sans-serif;
   color: #6070ff;
   font-weight: bold;
   font-size: 18px;
 }
 
 .name {
   width: 85%;
-  margin-top: 20px;
+  margin: 20px 20px 0 24px;
+  color: #172b4d;
 }
 
 .desc {
-  margin-top: 14px;
+  margin: 16px 10px 20px 24px;
   font-weight: 400;
   width: 85%;
-  line-height: 1.5;
-  text-align: justify;
+  color: #344563;
+  line-height: 1.5em;
+}
+
+.connect {
+  width: 85%;
+  margin: 0 20px 15px 24px;
+  color: #7f8cff;
+  font-weight: 500;
 }
 
 .container-3 {
   display: flex;
+  margin: 0 20px 20px 24px;
 }
 
-.connect {
-  margin-top: 20px;
+/* Section Works */
+.works {
+  width: 100%;
+}
+
+.grid-1 {
+  width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: 640px 700px 640px 700px;
+  gap: 25px;
+}
+
+.work1 {
+  background-color: #fff;
+  border: solid 1px #dfe1e6;
+  border-radius: 16px;
+  margin-top: 30px;
+  margin-bottom: 30px;
+}
```

Comment:
As specified in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/933fcb5206004f4ecaf1a898c06fa616010cede4/portfolio/2_mobile.md?plain=1#L51), kindly ensure that for each project, the content of the project is aligned with the flexbox